nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -6,10 +6,13 @@ define([
     'common/busEventManager',
     'common/ui',
     'common/events',
+    'common/props',
     'base/js/namespace',
     'kb_common/html',
     './cellControlPanel',
-    './tabs/configure'
+    'common/cellComponents/tabs/infoTab',
+    './tabs/configure',
+    'json!./testAppObj.json'
```

Comment:
This is a mock of appCellData since we have not decided how we are going to work with this yet.